scripts/provisioning/s3_setup

```diff
@@ -44,7 +44,7 @@ def main():
 
   testCmd = subparsers.add_parser('test', help='setup test')
   testCmd.add_argument("--config", help='config URL', type=str, required=True)
-  testCmd.add_argument("--plan", help='test plan', type=str, required=False)
+  testCmd.add_argument("--plan", help='test plan', type=str, required=True)
```

Comment:
I guess we do not need to make this option as mandatory. @AjinkyaDhumal , @nileshgovande !!!

Reply:
Agree.

Reply:
Done